google/resource_compute_url_map.go

```diff
@@ -317,112 +321,63 @@ func resourceComputeUrlMapRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("self_link", urlMap.SelfLink)
 	d.Set("map_id", strconv.FormatUint(urlMap.Id, 10))
 	d.Set("fingerprint", urlMap.Fingerprint)
+	d.Set("default_service", urlMap.DefaultService)
 
-	hostRuleMap := make(map[string]*compute.HostRule)
```

Comment:
This is a breaking change. It changes the behavior for `host_rule`, `path_matcher` and `test`.

In the old behavior, if you had some entries created from outside Terraform, it would not create a diff. Now, it would show a diff. For proper import support however, I know this is required.

To be honest, it is probably a good thing. Better to manage everything in Terraform. However, since it is changing the behavior, we will need to wait longer before we release it. We do frequent release of non-breaking changes. And longer cycle (we wait until we have a few breaking changes) before doing a release with breaking changes. We do this to avoid the pain to our users of having to deal constantly with breaking changes requiring them to update their config.

The other 3 resources are good to go. Can you put the url map in a separate PR? I will merge the other 3 resources and then we will merge the url map separately when we are ready to do a breaking change release.

Thank you